### PR TITLE
fix: fixed utf8 decoding

### LINKED ISSUE #1: Python 3 issue
When running this plugin on Python 3 I am presented with the following:

```
Traceback (most recent call last):
  File "/home/venv/lib/python3.3/site-packages/errbot/errBot.py", line 484, in _execute_and_send
    reply = method(mess, match) if match else method(mess, args)
  File "/home/err-sktravel/sktravel.py", line 102, in mhd
    return self.send_output(nick, f, t, date, result=r)
  File "/home/err-sktravel/sktravel.py", line 42, in send_output
    out = unicode(out.strip(codecs.BOM_UTF8), 'utf-8')
NameError: global name 'unicode' is not defined
```

I believe  that `unicode` [no longer exists in Python 3](https://stackoverflow.com/questions/6812031/how-to-make-unicode-string-with-python3), and for a good reason.

### PR DESCRIPTION
**Pull request by Adman**
- switching to py3 unicode is not supported anymore
- fixes #1

Signed-off-by: Adrian Matejov a.matejov@centrum.sk

### CODE REVIEW

File: sktravel.py [hunk 1]
```diff
@@ -37,10 +38,12 @@ def send_output(self, nick, dep, dest, date, result=None):
                 time = obj.lines[0].departure
             elif hasattr(result[0], 'drives') and len(obj.drives) >= 1:
                 time = result[0].drives[0].begin_time
+                decode_msg = False
             searched[nick] = [dep, dest, time, date]
 
-        out = unicode(out.strip(codecs.BOM_UTF8), 'utf-8')
-        return out.encode('utf-8')
+        if decode_msg:
```
Comment (mrshu) [inline sktravel.py]: @Adman would it be possible to provide a of reasoning, as to why this works/is necessary?

**Comment by mrshu** [comment]
@adman looks fairly good in general -- I just do not understand the logic behind the fix.

File: sktravel.py (hunk 1 above)
Comment (Adman) [inline sktravel.py]: @mrshu imhdsk returns message like this `u"travel_info"` and `.decode` cannot be used. If you try it, exception `AttributeError: 'str' object has no attribute 'decode'` is thrown

Comment (mrshu) [inline sktravel.py]: @Adman looks like this should be fixed on another level then -- in `imhdsk`, right?

Comment (Adman) [inline sktravel.py]: @mrshu alright, I could then remove this `decode_msg` variable

**Comment by mrshu** [comment]
Much better @Adman, thanks!

**Pull request merged**